include/args.h

```diff
@@ -104,6 +104,17 @@ struct CreateMultipartUploadArgs : public ObjectArgs {
   utils::Multimap headers;
 };  // struct CreateMultipartUploadArgs
 
+struct ListMultipartUploadArgs : public BucketArgs{
```

Comment:
Practically we need [ListObjectsCommonArgs](https://github.com/minio/minio-cpp/blob/main/include/args.h#L178) to be extended, but overriding `max_keys` as `max_uploads`, then new fields `key-marker`/`upload-id-marker` need to be added

Reply:
Ok, so should I delete ListMultipartUploadArgs and pass that arguments to ListObjectsCommonArgs right?